### DISCUSSION TITLE
responsive grid for mobile, tablets, etc

### DISCUSSION THREAD

**Discussion by rhobinn** [grid-js/gridjs#650] [Ideas]
Are there plans to make it responsive?

right now the it loads it satys the same size after changing the browser size....

**Comment by afshinm** [comment]
Right now, the grid container correctly renders the horizontal scroll and headers if you resize your viewport. Could you let me know what you exactly mean by a responsive table? Do you have any examples?

**Reply by sabatale** [reply]
Here is an example (white-space: non-wrap;) in Edge where you:

1. Open the table with static vertical bar.
2. Minimize the bar.

https://user-images.githubusercontent.com/50220345/125364701-f2738080-e340-11eb-9e0d-9dafeb12d535.mp4

**Comment by loctran016** [comment]
Here is my [demo on StackBlitz](https://stackblitz.com/edit/js-ncwhrg). There is white-space (similar to the video above) when the screen is too large which looks annoying.
